src/OpenFeatureSDK/Model/EvaluationContext.cs

```diff
@@ -64,182 +77,27 @@ public class EvaluationContext
             return new Dictionary<string, Value>(this._structure.AsDictionary());

```

Comment:
Should we be returning am immutable dictionary?

Reply:
It would probably be for the best.

Reply:
Updated. It actually made things better. The structure is an immutable dictionary now, so it just flows through.